docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+- [Snowtrace Testnet Explorer](https://testnet.snowtrace.io/): To view the deployed smart contract.
+
+
+:::caution
+**DISCLAIMER: This Solidity smart contract tutorial is for demonstration purposes only. Users should consider proper precautions, error handling, and safeguards for production use. No one at Ava Labs is responsible for your development, and you must take full responsibility for ensuring your code is secure.**
+:::
+
+## Preparing Your NFT Files
+
+The first step of setting up an NFT smart contract is having your NFT files ready to use. In this example, the files will get uploaded to [Pinata](https://www.pinata.cloud/), a pinning service that prevents files from being garbage collected on IPFS.
+
+If you are unfamiliar with the process of uploading image and metadata files to an IPFS provider for NFT collection usage, please check out [this article](preparing-nft-files.md). Ensure that your files are uploaded and your base URI is ready to plug into your smart contract.
+
+Once the image and metadata files are ready, we can prepare to deploy a smart contract.
+
+## Preparing Your Environment
+
+### MetaMask Extension
+
+You will need the MetaMask Extension installed on whatever browser you are using to be able to fund the deployment of the smart contract. If you have not done so already, download MetaMask and [add the Fuji network to MetaMask](https://support.avax.network/en/articles/6224787-how-to-connect-metamask-to-the-fuji-testnet). Create or import a Fuji account as necessary.
+
+### Getting Testnet Funds
+
+Because we are deploying on the Fuji Network, you will need to get AVAX on the Fuji network. If you visit the [Avalanche Faucet](https://faucet.avax.network/), you can request up to 2 Fuji AVAX per day. Please enter the C Chain address of the account linked to your MetaMask in the previous step. 
+
+![Avalanche Faucet](intro-to-erc721s/1-faucet.png)
+
+## Creating the Smart Contract
+
+To create the smart contract, we are going to use [Open Zeppelin](https://docs.openzeppelin.com/). Open Zeppelin is a key tool for building smart contracts quickly and easily. While we are only scratching the surface in this tutorial, ample documentation is available on their website for you to read when you want to build more complex contracts.
+
+Open Zeppelin provides a [Contract Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard) that will build out ERC contracts. To avoid any complex coding environments, we will use this to create our ERC721 contract.
+
+![Contract Wizard](intro-to-erc721s/2-contract-wizard.png)
+
+Select `ERC721` on the Contract Wizard to get started. This will create the contract in the [Solidity programming language](https://docs.soliditylang.org/en/v0.8.15/).
+
+As you can see, the template contract is bare-boned. We will fill out the information in the left panel to auto-populate it into our contract. Make sure you change to the ERC721 tab as you get started to make the proper contract.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "We'll modify here..." instead of "We will modify here..."